.drone.yml

```diff
@@ -865,11 +869,32 @@ steps:
   depends_on:
   - installer
 
+- name: image-snap-aws
```

Comment:
adding a separate step for now to not clobber existing aws workflow.

Reply:
Have you gotten the import to finish successfully and have you been able to boot a node? If we can get this working, let's drop the packet stuff entirely and replace it with this. This will align AWS with how we are doing GCP and Azure.